File: EzDeal/API JAVA/src/main/java/com/myorg/ezdeal/models/TipoServicio.java
package com.myorg.ezdeal.models;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.io.Serializable;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name="tipo_servicio")
public class TipoServicio implements Serializable {

    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name="tipo_servicio_id")
    private Long id;

    @Column(name="nombre")
    private String nombre;

    @Column(name="descripcion")
    private String descripcion;

}

File: EzDeal/API .NET/EzDeal.Domain/TipoServicio.cs
using System.Collections.Generic;
namespace EzDeal.Domain
{
    public class TipoServicio
    {
        public int Id {get; set;}
        public string Nombre {get;set;}
        public string descripcion {get; set;}
        public ICollection<Servicio> Servicios {get; set;}

    }
}